reciprocalspaceship/__init__.py

```diff
@@ -15,6 +15,9 @@ def getVersionNumber():
 # Import algorithms submodule
 from reciprocalspaceship import algorithms
 
+# Import stats submodule
+from reciprocalspaceship import stats
```

Comment:
Let's remove this from the top-level API -- as discussed offline, we can add a method to `DataSet` that wraps this call.